jsonpaths/com.delighted/survey_response_created_1.json

```diff
@@ -0,0 +1,40 @@
+{
+    "jsonpaths": [
+        "$.schema.vendor",
+        "$.schema.name",
+        "$.schema.format",
+        "$.schema.version",
+        "$.hierarchy.rootId",
+        "$.hierarchy.rootTstamp",
+        "$.hierarchy.refRoot",
+        "$.hierarchy.refTree",
+        "$.hierarchy.refParent",
+        "$.data.event_data.comment",
+        "$.data.event_data.created_at",
+        "$.data.event_data.id",
+        "$.data.event_data.notes",
+        "$.data.event_data.permalink",
+        "$.data.event_data.person.created_at",
+        "$.data.event_data.person.email",
+        "$.data.event_data.person.id",
+        "$.data.event_data.person.name",
+        "$.data.event_data.person.phone_number",
+        "$.data.event_data.person_properties.Browser",
+        "$.data.event_data.person_properties.Country",
+        "$.data.event_data.person_properties.Device Type",
```

Comment:
I am sure there's a good reason for going with these property names, but the spaces make me very nervous. How difficult would it be to conform them to the rest of the fields, like `device_type` or `user_agent`?

Reply:
At the moment unfortunately the webhook sends these fields through with spaces so without an adapter we're stuck with the spaces.